### ISSUE TITLE
[Bug][Backend] The devlake backend started with error: chdir /app/python/plugins: no such file or directory

### ISSUE THREAD

**Issue by lshmouse** [apache/incubator-devlake#4764]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar issues.

### What happened

The devlake backend started with error: chdir /app/python/plugins: no such file or directory

```
time="2023-03-24 17:31:25" level=info msg="endpoint GET /plugins/bamboo/connections/:connectionId/remote-scopes github.com/apache/incubator-devlake/server/api.handlePluginCall.func1 5"
[GIN-debug] [WARNING] You trusted all proxies, this is NOT safe. We recommend you to set a value.
Please check https://pkg.go.dev/github.com/gin-gonic/gin#readme-don-t-trust-all-proxies for details.
[GIN-debug] Listening and serving HTTP on 0.0.0.0:8080
time="2023-03-24 17:31:25" level=error msg="chdir /app/python/plugins: no such file or directory
 Wraps: (2) chdir /app/python/plugins
 Wraps: (3) no such file or directory
 Error types: (1) *hintdetail.withDetail (2) *fs.PathError (3) syscall.Errno"
[GIN] 2023/03/24 - 17:31:44 | 200 |    3.560415ms |   172.20.33.152 | GET      "/blueprints"
[GIN] 2023/03/24 - 17:31:51 | 200 |     217.623µs |    172.18.2.165 | GET      "/version"
```

### What do you expect to happen

The devlake backend start  successfully without error~

### How to reproduce

Using devlake helm chart v0.16.0-beta10 to deploy devlake with build-in mysql database on k8s 

### Version

main

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by abeizn** [comment]
Thanks for reporting the issue,we will take a look at it.

**Comment by keon94** [comment]
This is a benign error message that doesn't disrupt the server's operation. It's been fixed in #4734 which will be in v0.17. @hezyin should we patch it back to v0.16?
